### PR TITLE
Vapid authentication

### PR DESCRIPTION
**Pull request by arjan**
Based on the [web-push javascript vapid implementation](https://github.com/web-push-libs/web-push/blob/master/src/vapid-helper.js), I added VAPID authentication, which works on modern Chrome and Firefox browsers. 

This fixes #1.

### CODE REVIEW

**Comment by arjan** [comment]
I see that this PR is not in good shape, I'll update the PR soon

File: lib/web_push_encryption/push.ex [hunk 1]
```diff
@@ -28,15 +30,21 @@ defmodule WebPushEncryption.Push do
...
+        "TTL" => "0",
+        "Content-Encoding" => "aesgcm",
+        "Encryption" => "salt=#{ub64(payload.salt)}",
+      })
+
+    headers = headers |> Map.put("Crypto-Key", "dh=#{ub64(payload.server_public_key)};" <> headers["Crypto-Key"])
 
     {endpoint, headers} = make_request_params(endpoint, headers, auth_token)
 
+    IO.inspect headers, label: "headers"
```
Comment (danhper) [inline lib/web_push_encryption/push.ex]: I suppose this was for debugging purpose?

**Review by danhper — CHANGES_REQUESTED** [review]
Thank you very much for the PR.
It seems you forgot a `IO.inspect`, otherwise LGTM!

**Comment by arjan** [comment]
You're right, that was a leftover. Fixed it!

**Comment by danhper** [comment]
Thank you very much!

**Comment by arjan** [comment]
Awesome, would you mind releasing a new version to hex.pm?

**Comment by danhper** [comment]
Just released as 0.2.0, thanks again for the PR!

**Pull request merged**